suse-module-tools.spec

```diff
@@ -99,6 +101,16 @@ This package contains the legacy "weak-modules" script for kernel
 module package (KMP) support. It was replaced by "weak-modules2" in
 SLE 11 and later. It is still used by the DKMS module packaging framework.
 
+%if 0%{?suse_version} >= 1550
+%package -n suse-kernel-rpm-scriptlets
+Summary:        Traditional kernel scriptlets
+Requires:       suse-module-tools = %{version}
+Provides:       suse-module-tools:/usr/lib/module-init-tools/kernel-scriptlets
+
+%description -n suse-kernel-rpm-scriptlets
+Traditional kernel scriptlets
```

Comment:
"traditional" is too subjective in my opinion. Please change this to a neutral wording.